core/domain/rights_manager.py

```diff
@@ -242,7 +323,18 @@ def create_new_exploration_rights(exploration_id, committer_id):
 
 
 def get_exploration_rights(exploration_id, strict=True):
-    """Retrieves the rights for this exploration from the datastore."""
+    """Retrieves the rights for this exploration from the datastore.
+
+    Args:
+        exploration_id: str. ID of exploration.
+        strict: bool. Whether to raise an error when ID is not found.
+
+    Returns:
+        ActivityRights. Rights associated to the exploration.
```

Comment:
Changed it to "The rights object for the given exploration." following the conventions.